riotfile.py

```diff
@@ -1644,6 +1644,25 @@ def select_pys(min_version=MIN_PYTHON_VERSION, max_version=MAX_PYTHON_VERSION):
                 ),
             ],
         ),
+        Venv(
+            name="unittest",
+            command="pytest --no-ddtrace {cmdargs} tests/contrib/unittest_plugin/"
+            + " --ignore=tests/contrib/unittest_plugin/snapshot/",
+            pkgs={"msgpack": latest},
+            env={"DD_CIVISIBILITY_UNITTEST_ENABLED": "1"},
+            pys=select_pys(min_version="2.7", max_version="3.11"),
+        ),
+        Venv(
+            name="unittest-snapshot",
+            command="pytest --no-ddtrace {cmdargs} tests/contrib/unittest_plugin/snapshot/",
+            pkgs={"msgpack": latest},
+            env={
+                "DD_CIVISIBILITY_UNITTEST_ENABLED": "1",
+                "DD_TRACE_AGENT_URL": "http://localhost:9126",
+                "AGENT_VERSION": "testagent",
+            },
+            pys="3.7",
+        ),
```

Comment:
We shouldn't need this. Running the circleci job with `snapshot: True` configuration should set the ddtrace agent url to the expected value.